R/waterfall_buildMain.R

```diff
@@ -84,62 +82,36 @@ waterfall_buildMain <- function(data_frame, grid=TRUE, label_x=FALSE,
 
     # Theme, Boolean, if specified to plot x labels, define theme such that
     # labels are plotted
-    if(label_x == TRUE & plot_x_title == TRUE)
-    {
-        theme <-  theme(axis.ticks=element_blank(),
+    default_theme <- theme(axis.ticks=element_blank(),
```

Comment:
This function is easier to maintain if duplicated theme elements are separated out (easier to change defaults, etc). Similarly, you could probably move some duplicated elements of the if/else below such that each option is only specified once